migrate/live-migration/live-migration-from-postgres.md

```diff
@@ -0,0 +1,252 @@
+  -f roles.sql \
+  -f dump.sql
+```
+
+## 4. Convert hypertables and enable Timescale features
+
+This is the ideal point to convert regular tables into hypertables. This step
+is optional. It can even be performed after all the data has been migrated,
+since hypertables can be created from tables with existing data.
+
+The advantage of converting before inserting the data is that the data is
+partitioned as it's written into the database. If done afterwards, the data
+has to first be written, then moved to the corresponding partition when the
+table is converted into a hypertable.
+
+A more detailed explanation can be found in the [hypertable
+documentation][hypertable-docs].
+
+In simple terms, you'll want to convert the tables that contain time series
+data. For each table that's going to be converted to a hypertable in the target
+database, run the following command:
+
+```sh
+psql -X -d "$TARGET" \
+  -v ON_ERROR_STOP=1 \
+  -c "SELECT create_hypertable('<table name>', '<time column name>')"
+```
+
+Once the table is converted, you can follow the guides to enable more Timescale
+features like [retention][retention-docs], [compression][compression-docs], and
+[data tiering][data-tiering-docs].
+
+## 5. Migrate initial data from source to target
+
+Use `pgcopydb` and the snapshot ID obtained in the second step to copy the
+data. The [snapshot][snapshot] ID is exported to `${TMPDIR}/pgcopydb/snapshot`.
+If `$TMPDIR` is not set, it's exported to `/tmp/pgcopydb/snapshot`.
+
+```sh
+pgcopydb copy table-data \
+  --source "$SOURCE" \
+  --target "$TARGET" \
+  --snapshot $(cat /tmp/pgcopydb/snapshot) \
+  --split-tables-larger-than <same-table concurrency size threshold>  \
+  --table-jobs <jobs: usually the number of core in SOURCE>
+```
+
+## 6. Apply the replication changes
+
+With the schema and initial data loaded, you can now apply the buffered and
+live changes emitted by the replication slot.
+
+```sh
+pgcopydb stream sentinel set apply --source $SOURCE
+```
+
+The replication progress can be monitored by querying the [pg_stat_replication
+view][pg-stat-replication-docs]:
+
+```sh
+psql $SOURCE \
+  -f - <<'EOF'
+SELECT replay_lag
+FROM pg_stat_replication
+WHERE application_name='pgcopydb' AND state='streaming'
+EOF
+```
+
+- `replay_lag`: Time elapsed between flushing recent WAL locally and receiving
+  notification that this standby server has written, flushed and applied it.
+
+Before proceeding to the next step, you need to wait until the replication lag
+is minimized as much as possible. For instance, you can move forward when the
+`replay_lag` is close to just a few seconds.
+
+## 7. Promote target database as new primary
+
+At this point, the source and target should be almost identical, with the
+`replay_lag` close to just a few seconds.
+
+When it's time to perform the switchover, stop the write traffic to the source
+database.
+
+Since no new data is being ingested, `pgcopydb` can be instructed to stop the
+replication process.
```

Comment:
I think that we need to be cautious here. If the replication lag _doesn't_ go down to a few seconds, then the user _must_ wait for it to go down here. We don't mention this, and it could lead to data loss, which we don't want.